jsgrad/env/web.ts

```diff
@@ -62,32 +62,53 @@ export class WebEnv {
   tempFile = async (): Promise<string> => `/tmp/${(Math.random() * 100000000).toFixed(0)}`
   mkdir = async (path:string): Promise<void> => {}
   fetchSave = async (url: string, path: string, dir?: string, onProgress?: TqdmOnProgress) => {
-    path = this.realPath(dir || "", path)
+    path = this.realPath(dir || '', path)
     const cache = await this._cache()
-    const cached  = await cache.match(path)
-    if (cached) return path
+    const cached = await cache.match(path)
+    if (cached) {
+      if (vars.DEBUG >= 2) console.log(`Cache hit for ${path}`)
```

Comment:
it's a workaround for an error i'm getting locally when install whisper encoders (tokenizer). I will revisite this later